fuse_constraints/include/fuse_constraints/marginal_constraint.h

```diff
@@ -122,13 +121,20 @@ class MarginalConstraint : public fuse_core::Constraint
    */
   const std::vector<fuse_core::VectorXd>& x_bar() const { return x_bar_; }
 
+#if !CERES_SUPPORTS_MANIFOLDS
   /**
    * @brief Read-only access to the variable local parameterizations
    */
   const std::vector<fuse_core::LocalParameterization::SharedPtr>& localParameterizations() const
   {
     return local_parameterizations_;
   }
+#else
+  /**
+   * @brief Read-only access to the variable local parameterizations
```

Comment:
Pendantic, but this comment should be updated:
```suggestion
   * @brief Read-only access to the variable manifolds
```